src/modules/auth/graphql/types.rs

```diff
@@ -2,6 +2,12 @@ use async_graphql::{Enum, SimpleObject, ID};
 use chrono::{DateTime, Local};
 use serde::{Deserialize, Serialize};
 
+use crate::modules::user::model::User as UserModel;
+
+// enum Role {
+//     User,
+// }
+
```

Comment:
Feel free to reactivate this in a separate PR, but you may want to have a `Enum` trait derived.

https://async-graphql.github.io/async-graphql/en/define_enum.html